src/Documentation/fpga/xrt.rst

```diff
@@ -471,6 +471,27 @@ these subsystems will be handled in leaf drivers with root and partition drivers
 being part of the infrastructure and provide common services for all leaves found
 on all platforms.
 
+The driver object model looks like the following::
```

Comment:
you can remove one extra ':'

Reply:
The :: makes RST treat it as monospace/code and draw a box around it.